appveyor.yml

```diff
@@ -27,43 +39,85 @@ install:
   - 7z x mariadb-%MARIADB_VERSION%.tar > nul
   - cd mariadb-%MARIADB_VERSION%
   - rmdir /S /Q storage\mroonga\
+  - ps: Get-Content ..\mroonga\version | ForEach-Object { $env:MROONGA_VERSION = $_ }
   - move ..\mroonga storage\mroonga
   - git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
+  - ps: Get-Content ..\groonga\base_version | ForEach-Object { $env:GROONGA_VERSION = $_ }
+  - ps: Get-Content ..\groonga\include\groonga\version.h.in | ForEach-Object { $_ -replace "@GRN_VERSION@","$env:GROONGA_VERSION" } | Set-Content ..\groonga\include\groonga\version.h
   - rmdir /S /Q ..\groonga\test\
   - cd ..\groonga\vendor
-  - c:\Ruby22-x64\bin\ruby -v download_lz4.rb
-  - c:\Ruby22-x64\bin\ruby -v download_mecab.rb
+  - c:\Ruby26-x64\bin\ruby -v download_lz4.rb
+  - c:\Ruby26-x64\bin\ruby -v download_mecab.rb
   - cd ..\..\mariadb-%MARIADB_VERSION%
   - mkdir storage\mroonga\vendor
   - move ..\groonga storage\mroonga\vendor\groonga
   - git clone --quiet --depth 1 https://github.com/groonga/groonga-normalizer-mysql.git storage\mroonga\vendor\groonga\vendor\plugins\groonga-normalizer-mysql
-build_script:
-  - "echo # > win\\packaging\\CMakeLists.txt"
-  - cmake . -G "%CMAKE_GENERATOR_NAME%"
-      -DCMAKE_BUILD_TYPE=Debug
-      -DPLUGIN_ARCHIVE=NO
-      -DPLUGIN_BLACKHOLE=NO
-      -DPLUGIN_CASSANDRA=NO
-      -DPLUGIN_CONNECT=NO
-      -DPLUGIN_CSV=NO
-      -DPLUGIN_EXAMPLE=NO
-      -DPLUGIN_FEDERATED=NO
-      -DPLUGIN_FEDERATEDX=NO
-      -DPLUGIN_HEAP=NO
-      -DPLUGIN_INNOBASE=YES
-      -DPLUGIN_MYISAM=NO
-      -DPLUGIN_MYISAMMRG=NO
-      -DPLUGIN_OQGRAPH=NO
-      -DPLUGIN_PERFSCHEMA=NO
-      -DPLUGIN_SEQUENCE=NO
-      -DPLUGIN_SPHINX=NO
-      -DPLUGIN_SPIDER=NO
-      -DPLUGIN_TEST_SQL_DISCOVERY=NO
-      -DPLUGIN_TOKUDB=NO
-      -DPLUGIN_XTRADB=NO
-      -DWITH_UNIT_TESTS=OFF
-      -DWITH_MARIABACKUP=OFF
-      -DGRN_WITH_BUNDLED_MECAB=ON
-  - cmake --build . --config Debug
-
 test: off
+
+for:
+  -
+    matrix:
+      only:
+        - configuration: Debug
+    build_script:
+      - "echo # > win\\packaging\\CMakeLists.txt"
+      - cmake . -G "%CMAKE_GENERATOR_NAME%"
+          -DCMAKE_BUILD_TYPE=Debug
```

Comment:
I recommend using RelWithDebInfo in CMAKE_BUILD_TYPE. Because it is optimized and generate debug symbol.
Debug does not optimize so that slow.

Therefore, we should also make configuration of RelWithDebInfo.

Reply:
I think Debug is enough in this context because this configuration is used for finding build specific issues.
But, in point view of building package, it is better to use same configuration (RelWithDebInfo) because it can reduce tths risk to overloook build error with RelWithDebInfo specific issue.
So I'll fix it.